V1.9.5

Siel opened the pull request:
Goal: provide the user with the possibility of update gfortran within Pmetrics.

This should close #4

Siel committed:
#4 getGfortran changet to getOrUpdateGfortran, it will chen on linux and MacOs the current version of gcc and will update it, EXPERIMENTAL FEATURE this has not been tested yet.

Siel committed:
#4 getGfortran changet to getOrUpdateGfortran, it will chen on linux and MacOs the current version of gcc and will update it, EXPERIMENTAL FEATURE this has not been tested yet.

Siel committed:
avoid false registrations from development

Siel committed:
avoid false registrations from development

Siel committed:
Function exported

Siel committed:
Function exported

Siel commented on R/getGfortran.R:
```diff
@@ -1,4 +1,20 @@
+#' @author Michael Neely
+#' @export
+
+updateGfortran <- function() {
```
@mnneely What do you think about this function name?

mnneely replied:
It seems like the underscore is becoming the convention for naming variables, functions, etc. in R?

```suggestion
update_gfortran <- function() {
```

Siel replied:
Ok 👍

Siel replied:
Should it have a prefix? like PMupdate_gfortran?

mnneely replied:
Here's a nice coding convention reference.  As you can tell, I developed things a long time ago. I would not suffix items as op.1 now.  I would call it op_1.
https://www.r-bloggers.com/🖊-r-coding-style-guide/

Sabryr committed:
When using ifort on unix, correct syntax for syntax-only

Sabryr committed:
When using ifort on unix, correct syntax for syntax-only

Sabryr committed:
Commnet added for ifort syntax change , Fixes #6

Sabryr committed:
Commnet added for ifort syntax change , Fixes #6

mnneely committed:
Merge pull request #7 from Sabryr/syntaxcheck-fix-unix

Syntaxcheck fix unix

Siel committed:
version number

Siel committed:
version number

Siel committed:
Supporting windows installGfortran

Siel committed:
Supporting windows installGfortran

Siel committed:
function renamed

Siel committed:
function renamed

Siel commented:
Now we are supporting install_gfortran in both Mac and windows, linux would be a little bit harder because the commands will depend on the package manager the user is using.

I also included the script to remove the gfortran installation from the system. but I'm not sure how to proceed. should we also uninstall choco/brew ? @mnneely

mnneely commented:
The removal is to remove other gfortran installations?  I don't think we need to remove choco/brew.

Siel commented:
No, I was thinking about the user that want to uninstall all the dependencies pmetrics uses to setup gfortran

mnneely commented:
So install gfortran, but then remove choco/brew.  But then they won’t be able to update gfortran.

I guess we could warn them of that fact...

___________________________________________
Michael Neely, MD, MSc, FCP
Chief, Infectious Diseases
Director, Laboratory of Applied Pharmacokinetics and Bioinformatics
Children’s Hospital Los Angeles

Professor of Pediatrics and Clinical Scholar
Keck School of Medicine
University of Southern California

On May 5, 2020, at 3:50 PM, Julian Otalvaro <notifications@github.com> wrote:

﻿

No, I was thinking about the user that want to uninstall all the dependencies pmetrics uses to setup gfortran

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/LAPKB/Pmetrics/pull/5*issuecomment-624347071__;Iw!!LIr3w8kk_Xxm!8-6hc-KGt09Z-mupDFmW9NDeym8BJ7ahrdrGD1JlOI8wskA5f16x2C9C0A3sOA$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AB6CLTVV2GUKSIOFESCD6GDRQCJ3NANCNFSM4MKDLB7A__;!!LIr3w8kk_Xxm!8-6hc-KGt09Z-mupDFmW9NDeym8BJ7ahrdrGD1JlOI8wskA5f16x2C8GJInsyA$>.

mnneely committed:
Patch for SIMparse due to broken parallel function in Rstudio with R 4.0.0 update.

mnneely committed:
Sync local

Merge branch 'master' of github.com:LAPKB/Pmetrics

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

mnneely committed:
Patch for SIMparse due to bug in Rstudo with parallel function in R 4.0.0

Siel committed:
Merge branch 'v1.9.4' of github.com:LAPKB/Pmetrics into v1.9.4

Siel committed:
branch promoted to v1.9.4 given an unplaned hotfix

Siel committed:
fixing an error in Linux, closes #10

Siel committed:
v1.9.5

Siel committed:
v1.9.5

Siel committed:
fixing last commit

Siel committed:
updating NEWS

Siel committed:
the css is backed up at the lapk.ork server, now users can see the report /w styles also if they dont have pmetrics installed (remote execution)

mnneely committed:
fixed bug if >1 output equation

mnneely committed:
Merge pull request #14 from LAPKB/Issue_11

fixed bug if >1 output equation
